aerogear-android-push/src/main/java/org/jboss/aerogear/android/unifiedpush/gcm/AeroGearGCMPushRegistrar.java

```diff
@@ -159,6 +168,11 @@ protected Exception doInBackground(Void... params) {
                         postData.addProperty("variantId", variantId);
                         postData.addProperty("secret", secret);
                         presistPostInformation(context.getApplicationContext(), postData);
+                        GcmPubSub gcmPubSub = gcmPubProvider.get(context);
+                    
+                        for (String catgory : categories) {
```

Comment:
We could... _always_ do that :) autosubscribe to the `/topics/variantId` topic :-) 

but we should not add the variantID to the category metadata ;-)